lib/Ohmbrewer_Thermostat.cpp

```diff
@@ -1,93 +1,99 @@
 #include "Ohmbrewer_Thermostat.h"
 #include "Ohmbrewer_Screen.h"
 #include "Ohmbrewer_Publisher.h"
+#include "pid.h"
 
+//Define Variables we'll be connecting to with PID
+double setPoint;
+double input;
+double output;
 
+//Define the aggressive and conservative Tuning Parameters for PID
+double aggKp=4;
+double aggKi=0.2;
+double aggKd=1;
+double consKp=1;
+double consKi=0.05;
+double consKd=0.25;
 
-/**
- * The desired target temperature. Defaults to Celsius
- * @returns The target temperature in Celsius, as a Temperature object pointer
- */
-Ohmbrewer::Temperature* Ohmbrewer::Thermostat::getTargetTemp() const {
-    return _targetTemp;
-}
-
-/**
- * Sets the target temperature
- * @param targetTemp The new target temperature in Celsius
- * @returns The time taken to run the method
- */
-const int Ohmbrewer::Thermostat::setTargetTemp(const double targetTemp) {
-    unsigned long start = millis();
-
-    _targetTemp->set(targetTemp);
+//PID windowing variables
+int windowSize = 5000;              //changeable value
```

Comment:
Alternatively, if you're just trying to capture the default values to hand in to the PID object *and* you're going to have a PID object in the Thermostat anyhow, you can set those values up as either static class variables or a struct. For the former, check out how I declared the named colors in Screen.